jira/client.py

```diff
@@ -92,7 +92,8 @@ def emit(self, record):
 #    "which means that there is a big change of having problems. " \
 #    "Possible workaround http://stackoverflow.com/a/17628350/99834" % encoding)
 
-logging.getLogger('jira').addHandler(NullHandler())
+logger = logging.getLogger('jira')
```

Comment:
@vkrizan pointed about `__package__` which seems like the winner even if almost none is using. Out of curiosity I added [this question on SO](http://stackoverflow.com/questions/41694787/how-should-i-name-the-python-logger-inside-a-python-library).